### DISCUSSION TITLE
Can you make a Pressed_for(time) instance?

### DISCUSSION THREAD

**Discussion by Rimbaldo** [bxparks/AceButton#98] [Ideas]
Hi! 

Would it be possible to have a **Pressed_for (time)** instance, so we could compare between a same button being pressed for 2 seconds (do something), still pressed now for  4 seconds, do something else?

Regards, Rodrigo

**Comment by bxparks** [comment]
How about setting:

* setLongPressDelay(2000)
* setRepeatPressDelay(2000)
* setRepeatPressInterval(some large value, e.g. 2000, doesn't matter)

Then you handle the `kLongPressed`, the first `kRepeatPressed`, then ignore all the subsequent `kRepeatPressed`?
